.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.8, "3.12"]
+        python-version: [3.9, "3.12"]
```

Comment:
I think this leaves an orphaned 3.8 job on the PR but we can ignore than when merging.

Reply:
which one? All the tests seem 3.9 or 3.12 (as far as I can tell)

Reply:
The branch protection rules were expecting a 3.8 test so it didn't show this PR as ready to merge. I've manually fixed them at 3.9 and 3.12.